target_s3_jsonl/s3.py

```diff
@@ -43,10 +48,20 @@ def create_client(config):
     else:
         aws_session = boto3.session.Session(profile_name=aws_profile)
 
-    if aws_endpoint_url:
-        return aws_session.client('s3', endpoint_url=aws_endpoint_url)
-    else:
-        return aws_session.client('s3')
+    # config specifies a particular role to assume
+    # we create a session & s3-client with this role
+    if role_arn:
```

Comment:
This could perhaps be moved as an `elif role_arn` [here](https://github.com/ome9ax/target-s3-jsonl/blob/83b5b706bbfce763d0f6a4ca9f267bbd2e41617c/target_s3_jsonl/s3.py#L47)?